app/views/issues/_relevant_issues.html.slim

```diff
@@ -0,0 +1,17 @@
+section.portfolio
+  .row
+    .heading.col-md-12
+      h3 Схожі звернення:
+    - @relevant_issues.each do |relevant_issue|
+      .col-sm-6.col-md-3
```

Comment:
No I didn't.
I wrote: moved code blocks with "each" in views *into separate partials...
I didn't replace them with partials. Should I leave them as they're or fix something?